server/src/lib/db/mysql.ts

```diff
@@ -67,8 +71,8 @@ export default class Mysql {
   }
 
   query(text: string, values: any[], callback: Function) {
-    this.pool.query(text, function (error: any, results: any, fields: any) {
-        error ? callback(error.message, null) : callback(null, results);
+    this.pool.query(text, function(error: any, results: any, fields: any) {
```

Comment:
point about code style (which apparently prettier doesn't care about): for lambdas like this i think we should prefer arrow-functions over anonymous `function`s, to avoid accidental misuse of `this` inside the lambda